resources/views/users/view.blade.php

```diff
@@ -451,7 +451,7 @@
         <div class="tab-pane" id="files_tab">
           <div class="row">
             <div class="col-md-12 col-sm-12">
-              <p>Allowed file types are png, gif, jpg, jpeg, doc, docx, pdf, xls, txt, lic, zip, and rar. Max upload size allowed is 128M.</p>
+              <p>{{ trans('admin/hardware/general.upload_filetypes_help') }}</p>
```

Comment:
I don't see that string in that language file tho? `{{ trans('admin/licenses/general.upload_filetypes_help') }}`, or it should be added to the `resources/lang/en/admin/general.php` file probably?

Ideally we'd make sure that's consistent across all of the file uploads, it was a dumb choice (by me) to put that into the licenses language file, since it's used all over.

Reply:
It is already in the lang folder actually, i didn't copy the whole path. But it should be good now.